Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-
+resolver = "2"

```

Comment:
Without adding this, `cargo install` throws the following warnings:
```text
warning: some crates are on edition 2021 which defaults to `resolver = "2"`, but virtual workspaces default to `resolver = "1"`
note: to keep the current resolver, specify `workspace.resolver = "1"` in the workspace root's manifest
note: to use the edition 2021 resolver, specify `workspace.resolver = "2"` in the workspace root's manifest
warning: some crates are on edition 2021 which defaults to `resolver = "2"`, but virtual workspaces default to `resolver = "1"`
note: to keep the current resolver, specify `workspace.resolver = "1"` in the workspace root's manifest
note: to use the edition 2021 resolver, specify `workspace.resolver = "2"` in the workspace root's manifest
```
Set the resolver to "2" based on [rust workspaces docs](https://doc.rust-lang.org/cargo/reference/workspaces.html#virtual-workspace).